src/varity/vcf_to_hgvs/protein.clj

```diff
@@ -637,7 +659,8 @@
       (let [{ppos :pos, pref :ref, palt :alt}
             (if-not (#{:no-effect :unknown :overlap-exon-intron-boundary} (:type pvariant))
               (common/apply-3'-rule pvariant (:ref-prot-seq seq-info))
-              pvariant)]
+              pvariant)
+            seq-info (merge seq-info options)]
```

Comment:
to use `prefer-extension-for-initial-codon-alt?` in `protin-extension` because `protein-extension` is called from `protein-indel`.